test/unit/workers/world_location_news_page_worker_test.rb

```diff
@@ -43,7 +43,7 @@ class WorldLocationNewsPageWorkerTest < ActiveSupport::TestCase
     end
 
     I18n.with_locale(:fr) do
-      world_location.update(title: "Le Royaume-Uni et la France")
+      world_location.update!(title: "Le Royaume-Uni et la France")
```

Comment:
Could this line be moved into the previous commit, as that's where it was first added?

Reply:
Think this one might also actually be an unnecessary change as well from when I was playing around, so will see if just taking this change out affects the test (I think not)

Reply:
Taken out now

Reply:
Yeah, I was just about to say that rubocop complains if you don't use the bang method and don't ever use the method's output.